src/components/search/Search.jsx

```diff
@@ -71,14 +60,23 @@ const Search = (props) => {
     )
   } else if (props.resultAuthority) {
     results = (
-      <QASearchResults key="search-results" />
+      <QASearchResults {...props} key="search-results" />
     )
   }
 
   return (
     <div id="search">
       <Header triggerEditorMenu={props.triggerHandleOffsetMenu} />
-      {alert}
+      { props.error
```

Comment:
Maybe we should make our own `Alert` component.

Reply:
https://github.com/LD4P/sinopia_editor/issues/1423